.travis.yml

```
@@ -0,0 +1,31 @@
+language: generic
+
+matrix:
+  include:
+    - os: linux
+      dist: xenial
+      sudo: required
+
+branches:
+  only:
+    - master
+
+services:
+  - docker
+
+install:
+  # starts Docker container
+  - cd $TRAVIS_BUILD_DIR/docker
+  - docker-compose up -d
+
+before_script:
+  - skip
+
+script:
+  # executes PHPUnit tests
+  - cd $TRAVIS_BUILD_DIR
+  - docker container exec -it KvnrValidator-php vendor/bin/phpunit -v --debug --stop-on-failure --stop-on-error
```

```suggestion
  - docker container exec -it KvnrValidator-php vendor/bin/phpunit -v --testdox --stop-on-failure --stop-on-error
```
I prefer testdox as we do on broca 😄